bff/test/Hosts.Tests/TestInfra/Retries/RetriableFact.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Duende Software. All rights reserved.
+// See LICENSE in the project root for license information.
+
+using Xunit.Sdk;
+
+namespace Duende.Hosts.Tests.TestInfra.Retries;
+
+[XunitTestCaseDiscoverer(
+    typeName: "Duende.Hosts.Tests.TestInfra.Retries.RetriableTestDiscoverer",
+    assemblyName: "Duende.Hosts.Tests"
+)]
+public class RetriableFact : FactAttribute
```

Comment:
Please rename this to RetryableFact.

Reply:
Neither word is common in dictionaries (I had to search as well!) 

The best I could find was that "Triable" is a legal term, while "Retryable" is sometimes used by other programming tools, like Java Spring. But that's also what looks right to me, fwiw.